packages/common-all/src/abTests.ts

```diff
@@ -138,4 +157,5 @@ export const CURRENT_AB_TESTS = [
   MEETING_NOTE_TUTORIAL_TEST,
```

Comment:
as per offline discussion, let's go ahead and remove everything related to `MEETING_NOTE_TUTORIAL_TEST`